defaults.yaml

```diff
@@ -10,7 +10,9 @@ conf:
   minion_1_name: k8s-minion-1
   minion_2_name: k8s-minion-2
   minion_3_name: k8s-minion-3
-  kubernetes_nodes_base_box_id: "bento/centos-7.4"
+  vagrant_provider: libvirt
+  kubernetes_nodes_base_box_id_virtualbox: "bento/centos-7.4"
```

Comment:
Nest the values, it's clearer. Example: `kubernetes_nodes_base_box_id.virtualbox`